src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -3580,6 +3624,8 @@ public String save() {
 
         if (editMode != null) {
             if (editMode.equals(EditMode.CREATE)) {
+                
+                System.out.print("adding files...");
```

Comment:
no System.outs!

Reply:
oops. I put those in for debugs and i usually search for them before checking in.